README.md

```diff
@@ -163,18 +164,17 @@ Keylime uses the Intel TPM2 software set to provide TPM 2.0 support.
 
 These can be installed using your package manager.
 
-On Fedora 32 (and greater):
+* On Fedora 32 (and greater):
 
-`sudo dnf install tpm2-tss tpm2-tools tpm2-abrmd'
+`sudo dnf install tpm2-tss tpm2-tools tpm2-abrmd`
 
-On Ubuntu Ubuntu 20 LTS:
+* On Ubuntu 20 LTS:
 
-`sudo apt-get install tpm2-tss tpm2-tools tpm2-abrmd'
+`sudo apt-get install tpm2-tss tpm2-tools tpm2-abrmd`
 
-You can also build the tpm2-tss software stack (available at
-https://github.com/tpm2-software/tpm2-tss) as well as the
-tpm2-tools utilities available at https://github.com/tpm2-software/tpm2-tools. See
-README.md in these projects for detailed instructions on how to build and install.
+You can also build the tpm2-tss software stack (available at [tpm2-tss](https://github.com/tpm2-software/tpm2-tss))
```

Comment:
If you're going to rewrite this paragraph, I think it would be better to just remove the parentheical, and just make the first usage of "tpm2-tss" and "tpm2-tools" be the links to their repositories.